libpod/define/container_inspect.go

```diff
@@ -62,6 +62,8 @@ type InspectContainerConfig struct {
 	SystemdMode bool `json:"SystemdMode,omitempty"`
 	// Umask is the umask inside the container.
 	Umask string `json:"Umask,omitempty"`
+	// Secrets are the secrets mounted in the container
+	Secrets []string `json:"Secrets,omitempty"`
```

Comment:
This feels like it should be an array of structs

Reply:
Is there anything we can report about a secret besides its name?

Reply:
doing a podman secret inspect returns its name, id, creation time, and driver. That's the only information available on a secret

Reply:
Is it worth including any of that when inspecting the container? Name and ID seem like things we'd want to have.

Reply:
Name and ID sounds good, name definitely because the secret would be mounted at /run/secrets/name, and id to verify if its the actual secret, not one that has been deleted and remade with the same name